app/models/hook.rb

```diff
@@ -0,0 +1,11 @@
+class Hook < ApplicationRecord
+  def self.catalog
+    ['announcements', 'badges', 'events', 'memberships', 'people', 'teams', 'users'].map do |k|
```

Comment:
I couldn't find a good way to grab all of the classes that implement a module - if anybody has any ideas, I'm all ears.